src/skills-builder/skills-builder-modal/select-preferences/CareerInterestCard.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  IconButton, Icon,
+} from '@edx/paragon';
+import { Close } from '@edx/paragon/icons';
+import { SkillsBuilderContext } from '../../skills-builder-context';
+import { removeCareerInterest } from '../../data/actions';
+
+const CareerInterestCard = ({ interest }) => {
+  const { dispatch } = useContext(SkillsBuilderContext);
+
+  return (
+    <div className="d-flex justify-content-between align-items-center pb-2 pr-2 pl-4 rounded shadow-sm">
+      <p className="pt-4">
+        {interest}
+      </p>
+      <IconButton
+        iconAs={Icon}
+        src={Close}
+        alt={`Remove ${interest}`}
```

Comment:
The `alt` here refers to alternative text, right? "Remove" here would be hardcoded English, right? Is there a way to handle this with our i18n capabilities?